tests/functional_tests/L2_NeMo_2_Export_TRT_LLM.sh

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020-2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+coverage run --branch -a --data-file=/workspace/.coverage --source=/workspace/nemo tests/collections/llm/test_hf_import.py \
+  --hf_model /home/TestData/nlp/megatron_llama/llama-ci-hf \
+  --output_path /tmp/nemo2_ckpt
+
+coverage run --branch -a --data-file=/workspace/.coverage --source=/workspace/nemo tests/export/nemo_export.py \
+  --model_name test \
+  --model_type llama \
+  --checkpoint_dir /tmp/nemo2_ckpt \
+  --min_tps 1 \
+  --test_deployment True \
+  --run_accuracy True \
+  --test_data_path /tmp/lambada.json \
```

Comment:
Where this file is exactly created? We could use https://github.com/NVIDIA/NeMo/blob/main/tests/setup/data/create_sample_lambada.py to produce a tiny 5-sample Lambada dataset I prepared a while ago.

Reply:
Actually, need to disable the accuracy calculation since github tests are meant to be fast and this test will take several minutes.